onnxruntime/contrib_ops/rocm/bert/skip_layer_norm_tunable_op.h

```diff
@@ -55,6 +55,32 @@ Status SkipLayerNormSmallOp(const SkipLayerNormParams<T>* params) {
   return Status::OK();
 }
 
+#define ADD_OP(threads_per_block)                                         \
+  this->ops_.emplace_back(SkipLayerNormSmallOp<T, threads_per_block, 1>); \
+  this->ops_.emplace_back(SkipLayerNormSmallOp<T, threads_per_block, 2>); \
+  this->ops_.emplace_back(SkipLayerNormSmallOp<T, threads_per_block, 4>); \
+  this->ops_.emplace_back(SkipLayerNormSmallOp<T, threads_per_block, 8>); \
+  this->ops_.emplace_back(SkipLayerNormSmallOp<T, threads_per_block, 16>);
+
+template <typename T>
+class SkipLayerNormTunableOp : public TunableOp<SkipLayerNormParams<T>> {
+ public:
+  SkipLayerNormTunableOp() {
+    ADD_OP(32)
+    ADD_OP(64)
+    ADD_OP(128)
+    ADD_OP(192)
+    ADD_OP(256)
+    ADD_OP(320)
+    ADD_OP(384)
+
+    // NOTE: the 3-th kernel seems to be better in gerenal case, so set it as default one
+    this->SetDefaultId(3);
```

Comment:
I think we will need to add a non-small kernel to make sure there is always a kernel available for any input sizes. And we can make this kernel as default, so that there won't be cases where the kernel doesn't support certain input sizes.

Reply:
This PR only includes small case. I will change Defaultid in next pr which added normal case.